template/addons/next-auth/api-handler-prisma.ts

```diff
@@ -5,14 +5,15 @@ import CredentialsProvider from "next-auth/providers/credentials";
 // Prisma adapter for NextAuth, optional and can be removed
 import { PrismaAdapter } from "@next-auth/prisma-adapter";
 import { prisma } from "../../../server/db/client";
+import { env } from "../../../server/env";
```

Comment:
in your installers you copy this file to `server/env-schema` or am I missing something?

Reply:
I decided to split it into two separate files to keep it dry
- `env-schema.js` - contains the schama overwritten by the installers
- `env.js` - parses schema, formats errors and exports validated env variables